warehouse/stats.go

```diff
@@ -162,7 +164,17 @@ func (job *UploadJobT) recordTableLoad(tableName string, numEvents int64) {
 		pkgLogger.Errorf("[WH]: Failed to generate delay metrics: %s, Err: %v", job.warehouse.Identifier, err)
 		return
 	}
-	job.timerStat("event_delivery_time", tag{name: "tableName", value: strings.ToLower(tableName)}).SendTiming(time.Since(firstEventAt))
+
+	config := job.warehouse.Destination.Config
```

Comment:
Should we move this inside the if block of retried?

Reply:
Then in that case everyone don't need to do the syncFrequency retrieval from config.

Reply:
Moved inside. compared to actual uploads, retried will be minor